### PR TITLE
feat(config): add fontsource monorepo

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Add `fontsource` repository to list of monorepos

## Context:

Another PR was trying to add `fontsource` to a group, but it should be in the list of monorepos. (#10198)

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — DISMISSED** [review]

File: lib/config/presets/internal/monorepo.ts [hunk 1]
```diff
@@ -76,6 +76,7 @@ const repoGroups = {
   feathers: 'https://github.com/feathersjs/feathers',
   fimbullinter: 'https://github.com/fimbullinter/wotan',
   flopflip: 'https://github.com/tdeekens/flopflip',
+  fontsource: 'https://github.com/fontsource/fontsource',
```
Comment (viceice) [inline lib/config/presets/internal/monorepo.ts:79]:
```suggestion
  Fontsource: 'https://github.com/fontsource/fontsource',
```

😉

**Review by viceice — APPROVED** [review]

**Comment by HonkingGoose** [comment]
@viceice Can you explain why you use `Fontsource` while the other reponames use lowercase?

If there's some common rule, I could check the capitalization of the other names, so that we're consistent throughout Renovate bot. 😄

**Comment by viceice** [comment]
> @viceice Can you explain why you use `Fontsource` while the other reponames use lowercase?
> 
> If there's some common rule, I could check the capitalization of the other names, so that we're consistent throughout Renovate bot. 😄

Yes, They should follow the real name, renovate will slugify it automatically. but we shouldn't change existing, because it will /can break things.

**Comment by HonkingGoose** [comment]
> but we shouldn't change existing, because it will /can break things.

So if I do want to change it, it should be tagged `breaking` then, and go in a new major release?

**Pull request merged**